hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/virtual/IterableContractValue.java

```diff
@@ -366,6 +367,13 @@ public int getVersion() {
         return ITERABLE_VERSION;
     }
 
+    int getSerializedSize() {
+        int valueSize = uint256Value.length;
+        int prevSize = (prevUint256Key == null) ? 1 : 1 + prevUint256KeyNonZeroBytes;
```

Comment:
Same here adding a constant for 1 helps in `1 + prevUint256KeyNonZeroBytes`

Reply:
Done